### DISCUSSION TITLE
iLink draft build

### DISCUSSION THREAD

**Discussion by israpps** [israpps/wLaunchELF_ISR#29] [General]
what does this do? support browsing fat32 and EXFAT devices connected with special HDD case into the iLink port found on PS2 models ranging from `SCPH-10000` and `SCPH-39xxx`.

due to the lack of compatible iLink devices (f*ck you SBP2) I just made a rushed build and uploaded here after confirming with @hardlevel that it actually works

if you can: donate me an iLink case XD

known bugs or limits: 
- **MX4SIO** and **iLink** can't co-exist
- iLink won't work on PSX-DESR models (long story short: a brand new driver is needed for that IIRC)

<hr /><em>This discussion was created from the release <a href='https://github.com/israpps/wLaunchELF_ISR/releases/tag/ilink'>iLink draft build</a>.</em>

**Comment by Davidthegoliath333** [comment]
Idk if this is due to the version I downloaded but on the exfat ms4sio,  two memory cards can't be used simultaneously
